backend/pipeline/pipeline_processor.py

```diff
@@ -65,3 +65,22 @@ def update_pipeline_status(
             logger.error(f"Error occured while saving pipeline : {exc}")
             raise PipelineSaveError()
         return pipeline
+
+    @staticmethod
+    def update_pipeline(
+        pipeline_guid: Optional[str],
+        status: tuple[str, str],
+        is_active: Optional[bool] = None,
+    ) -> Any:
+        if not pipeline_guid:
+            return
+        check_active = True
```

Comment:
```suggestion
        check_active = not is_active
```

Reply:
we could replace the if case here as per code, please check
@chandrasekharan-zipstack